src/applications/disability-benefits/view-payments/actions/index.js

```diff
@@ -19,8 +22,15 @@ export const getAllPayments = () => async dispatch => {
     // TODO: fire off analytics event when endpoint is wired up.
     //   const errCode = res.errors[0].code;
     //   isServerError(errCode) ? recordEvent({}) : recordEvent({})
+    recordEvent({
+      event: `disability-view-dependents-load-failed`,
```

Comment:
This should be `view-payment-history-failed`

Reply:
duh, updated this